From 05b0c9e37de02a0f390383fa54a8f1ece654aa31 Mon Sep 17 00:00:00 2001
From: Joshua Strobl <joshua@streambits.io>
Date: Wed, 11 Mar 2020 23:13:51 +0200
Subject: [PATCH 1/1] Force GNOME_DESKTOP_BG_API_BREAK off.

We revert this ABI change to ensure Nautilus continues to compile.
---
 panels/background/cc-background-item.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/panels/background/cc-background-item.c b/panels/background/cc-background-item.c
index cb6bea28d..dc8850b66 100644
--- a/panels/background/cc-background-item.c
+++ b/panels/background/cc-background-item.c
@@ -156,11 +156,7 @@ render_at_size (GnomeBG *bg,
         GdkPixbuf *pixbuf;
 
         pixbuf = gdk_pixbuf_new (GDK_COLORSPACE_RGB, FALSE, 8, width, height);
-#ifdef GNOME_DESKTOP_BG_API_BREAK
-        gnome_bg_draw (bg, pixbuf);
-#else
         gnome_bg_draw (bg, pixbuf, gdk_screen_get_default (), FALSE);
-#endif
 
         return pixbuf;
 }
-- 
2.25.1

